tests/org.jboss.tools.openshift.test/src/org/jboss/tools/openshift/test/ui/models/DeployImageWizardModelTest.java

```diff
@@ -58,28 +58,28 @@ public void setUp() {
 		when(streamImport.getImageStatus()).thenReturn(Arrays.asList(status));
 	}
 	
-	@Test
-	public void testImageExistsLocally() {
-		model.setProject(null);
-		assertFalse(model.imageExistsLocally(null));
-		assertFalse(model.imageExistsLocally(" "));
-		
-		//no tag
-		model.imageExistsLocally("foo/bar");
-		verify(dockerConnection).hasImage("foo/bar", "latest");
-		
-		//has tag
-		model.imageExistsLocally("foo/bar:asf34fs");
-		verify(dockerConnection).hasImage("foo/bar", "asf34fs");
-		
-		//has registry+port and tag
-		model.imageExistsLocally("host:1234/foo/bar:asf34fs");
-		verify(dockerConnection).hasImage("host:1234/foo/bar", "asf34fs");
-		
-		//
-		model.setDockerConnection(null);
-		assertFalse(model.imageExistsLocally("foo/bar"));
-		
-	}
+//	@Test
```

Comment:
@fbricon right, it was meant to be removed because the `IDeployImagePageModel.imageExistsLocally` has been removed.